Homework-№4-from-28.09.2022/task 2/main.c

```diff
@@ -0,0 +1,30 @@
+#include "task2.h"
+
+int main()
+{
+    tests();
```

Comment:
```suggestion
    if (!tests())
    {
        printf("Tests failed!\n");
        return -1;
    }
```
Вот так нужно выполнять тесты.